Core/scripts/dirac-deploy-scripts.py

```diff
@@ -31,57 +31,61 @@
                   'dirac_platform.py']
 
 wrapperTemplate = """#!$PYTHONLOCATION$
+from __future__ import print_function, division
+
 #
-import os,sys,imp
+import os
+import sys
+import imp
 #
-DiracRoot = os.path.dirname(os.path.dirname( os.path.realpath( sys.argv[0] ) ))
+DiracRoot = os.path.dirname(os.path.dirname(os.path.realpath(sys.argv[0])))
 if 'DIRACPLAT' in os.environ:
   DiracPlatform = os.environ['DIRACPLAT']
 else:
-  platformPath = os.path.join( DiracRoot, "DIRAC", "Core", "Utilities", "Platform.py" )
-  with open( platformPath, "r" ) as platFD:
-    Platform = imp.load_module( "Platform", platFD, platformPath, ( "", "r", imp.PY_SOURCE ) )
+  platformPath = os.path.join(DiracRoot, "DIRAC", "Core", "Utilities", "Platform.py")
+  with open(platformPath, "r") as platFD:
+    Platform = imp.load_module("Platform", platFD, platformPath, ("", "r", imp.PY_SOURCE))
   DiracPlatform = Platform.getPlatformString()
   if not DiracPlatform or DiracPlatform == "ERROR":
-    print >> sys.stderr, "Can not determine local platform"
+    print("Can not determine local platform", file=sys.stderr)
```

Comment:
that could be executed in with python 2.6 no ? Is it compatible ?

Reply:
Yes:
```
-bash-4.1$ python2.6
Python 2.6.6 (r266:84292, Jul 16 2019, 10:08:25) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-23)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from __future__ import print_function
>>> print("Can not determine local platform", file=sys.stderr)
Can not determine local platform
```